tests/regression/nextcloud-rule-exclusions-plugin/9508131.yaml

```diff
@@ -0,0 +1,22 @@
+---
+meta:
+  author: "Esad Cetiner"
+  description: "Dav client scanning for Dav path"
+  enabled: true
+  name: 9508131.yaml
+tests:
+  - test_title: 9508131-1
+    stages:
+      - stage:
+          input:
+            dest_addr: 127.0.0.1
+            headers:
+              Host: localhost
+              User-Agent: OWASP CRS
```

Comment:
We are using `OWASP CRS test agent` as UA in the tests.

Reply:
Also, it would be best to explicitly specify the HTTP version using `version: "HTTP/1.1"` (for all tests).

Reply:
All tests should be updated now